docs/sources/setup-grafana/configure-grafana/configure-custom-branding/index.md

```diff
@@ -73,6 +73,9 @@ The configuration file in Grafana Enterprise contains the following options. Eac
 
 # Set to complete URL to override loading logo
 ;loading_logo =
+
+# Hides the Grafana edition being used in the footer if set to "true"
```

Comment:
```suggestion
# Set to `true` to remove the Grafana edition from appearing in the footer.
```

Reply:
I've used the text here but removed the full stop because none of the other configs have full stop in their comment either (this block comes directly from code)